pybossa/util.py

```diff
@@ -764,6 +762,26 @@ def generate_invitation_email_for_admins_subadmins(user, access_type):
                                   is_qa=is_qa)
     return msg
 
+def generate_notification_email_for_admins(user, admins_emails, access_type):
+
+    is_qa = current_app.config.get('IS_QA')
+    server_url = current_app.config.get('SERVER_URL')
+    admin_manual_label = current_app.config.get('ADMIN_MANUAL_LABEL')
```

Comment:
Is this necessary? The email is sent to current admins, so they should have received their manual already